index.html

```diff
@@ -171,5 +171,31 @@ <h3>Skills</h3>
                 </ul>
     </div>
 </section>
+<section class="form-flex">
+<p class="text-contact">I'm always interested 
+    in hearing about new projects, so if you'd 
+    like to chat please get in touch.
+</p>
+<form action="https://formspree.io/f/mnqynkkd" method="post" class="form">
+    <label for="Full name"></label>
+    <input class="inp-name" id="Full name" placeholder="Full name" type="text" maxlength="30">
+    <label for="Email address"></label>
+    <input class="inp-email" placeholder="Email address" type="email">
+    <label for="Enter text here"></label>
+    <textarea class="inp-text" placeholder="Enter text here" maxlength="500"></textarea>
+    <button class="btn-sub" type="submit">Get in touch</button>
```

Comment:
- [ ] You have shown a high level of understanding of the concepts by implementing the contact form 👏 👏 . However, the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#htmlcss-requirements) state that, the user must not be able to submit the form if one or more of the fields is empty. This is because improper validation of form data is one of the main causes of security vulnerabilities. It exposes your website to attacks such as header injections ☺️ 😌 . Kindly ensure to use [basic form validations](https://www.w3schools.com/tags/att_input_required.asp)